Classes/Backend/Preview.php

```diff
@@ -135,6 +135,7 @@ protected function attachAssets() {
 			$doc->backPath = $GLOBALS['BACK_PATH'];
 
 			$doc->getPageRenderer()->addCssFile($doc->backPath . ExtensionManagementUtility::extRelPath('flux') . 'Resources/Public/css/grid.css');
+			$doc->getPageRenderer()->addCssFile($doc->backPath . ExtensionManagementUtility::extRelPath('flux') . 'Resources/Public/css/icon.css');
```

Comment:
We don't need that line, was me mistake. The cache in the modules are really persistent.